### ISSUE TITLE
How to put a checkbox and text input horizontally 

### ISSUE THREAD

**Issue by zwz** [picturepan2/spectre#550]
Sometime we provide multiple choices to users, with a last `other` option for user to input their own answer. In this case, a checkbox with an input is required. When the checkbox is unchecked, the input is disabled. The code below
```
    <div class="input-group">
        <label class="form-checkbox">
            <input type="checkbox"><i class="form-icon"></i>
        </label>
        <input 
               class="form-input" type="text" placeholder="other answer" style="width:50px">
    </div>
```
puts the checkbox and the input horizontally for version 0.5.1. Now for version 0.5.7, it does not.

**Comment by picturepan2** [comment]
@zwz It works in v0.5.7. If you want the input disabled next to the unchecked checkbox. You can use JS to control it.

**Comment by zwz** [comment]
@picturepan2 Yes, the input is next to the checkbox, but the height of the input is much smaller than that of a normal one, as you can see below:
![2019-01-07 4 17 49](https://user-images.githubusercontent.com/897156/50756645-d5552700-1297-11e9-9f87-b965b68d960c.png)
